inspirehep/modules/migrator/cli.py

```diff
@@ -82,31 +150,30 @@ def populate(file_input=None,
              remigrate_broken=False,
              remigrate_all=False,
              wait=False):
-    """Populates the system with records from migrator files.
+    """DEPRECATED Populates the system with records from migrator files.
 
     Usage: inveniomanage migrator populate -f prodsync20151117173222.xml.gz
     """
     if remigrate_broken:
         click.echo("Remigrate broken records...")
-        remigrate_records.delay(only_broken=True)
+        migrate_from_mirror.delay(only_broken=True)
     elif remigrate_all:
         click.echo("Remigrate all records...")
-        remigrate_records.delay(only_broken=False)
+        migrate_from_mirror.delay(only_broken=False)
     elif file_input and not os.path.isfile(file_input):
         click.echo("{0} is not a file!".format(file_input), err=True)
     elif file_input:
         click.echo("Migrating records from file: {0}".format(file_input))
 
-        migrate(os.path.abspath(file_input), wait_for_results=wait)
+        migrate_from_file(os.path.abspath(file_input), wait_for_results=wait)
 
 
 @migrator.command()
 @click.option('--recid', '-r', type=int, help="recid on INSPIRE")
 def one(recid):
+    """DEPRECATED Migrate one record from legacy."""
```

Comment:
Maybe you could use a proper warning: https://docs.python.org/2/library/warnings.html